portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,10 +31,18 @@
 /** Servlet that returns some example content and handles data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> commentsStorage = new ArrayList<>();
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Do you have to create a new DatastoreService every time doGet() or doPost() is called? Or can you create it once and reuse it?

Reply:
Yes! The advice really makes sense, I put it in the beginning of this class for only once.